modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala

```diff
@@ -126,7 +125,7 @@ class packageSpec extends FlatSpec {
   it should "be in Running status after starting" in {
     val p = Process.eval(Task.now(start(seqId)))
     val qs = process(p)(qs1).take(1).runLast.unsafePerformSync.get._2
-    assert(qs.sequences(seqId).status === SequenceState.Running)
+    assert(qs.sequences(seqId).status == SequenceState.Running)
```

Comment:
😕 I tried the explicit types but still getting the same error. This really puzzles me because this PR didn't touch directly anything related to `SequenceState` and previously there was no issue. This is the concrete error:

```
[error] /Users/jnavarro/Develop/ocs3/modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala:128: type mismatch;
[error]  found   : edu.gemini.seqexec.model.Model.SequenceState
[error]  required: ?{def ===(x$1: ? >: edu.gemini.seqexec.model.Model.SequenceState): ?}
[error] Note that implicit conversions are not applicable because they are ambiguous:
[error]  both method ToEqualOps in trait ToEqualOps of type [F](v: F)(implicit F0: scalaz.Equal[F])scalaz.syntax.EqualOps[F]
[error]  and method convertToEqualizer in trait TripleEquals of type [T](left: T)packageSpec.this.Equalizer[T]
[error]  are possible conversion functions from edu.gemini.seqexec.model.Model.SequenceState to ?{def ===(x$1: ? >: edu.gemini.seqexec.model.Model.SequenceState): ?}
[error]     assert((qs.sequences(seqId).status: SequenceState) === (SequenceState.Running: SequenceState))
[error]                                       ^
[error] /Users/jnavarro/Develop/ocs3/modules/edu.gemini.seqexec.engine/src/test/scala/edu/gemini/seqexec/engine/packageSpec.scala:128: value === is not a member of edu.gemini.seqexec.model.Model.SequenceState
[error]     assert((qs.sequences(seqId).status: SequenceState) === (SequenceState.Running: SequenceState))
```